src/app/basic-form/basic-form-success.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+import {Location} from '@angular/common';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-basic-form-success',
+  templateUrl: './basic-form-success.component.html',
+  styleUrls: ['./basic-form.component.css'],
+})
+
+export class BasicFormSuccessComponent {
+    constructor(private fb: FormBuilder, private location: Location, private router: Router) { }
```

Comment:
Remove FormBuilder and Location use here, these aren't being used

Reply:
Done